src/components/Landing.js

```diff
@@ -0,0 +1,272 @@
+        title={headTitle}
+        titleTemplate={headTitleTemplate}
+        ogTitle={headOgTitle}
+        ogDescription={headOgDescription}
+        ogImage={headOgImgURLAbsolute}
+        keywords={headKeywords}
+        pageURL={absolutePageURL}
+      />
+      <Helmet>
+        <link rel="canonical" href={canonicalURL} />
+      </Helmet>
+      <AppBar color="primary" position="sticky">
+        <Toolbar>
+          <div className={cx.logoContainer}>
+            <div
+              data-test-id="logo-container"
+              style={{
+                display: 'flex',
+                justifyContent: 'flex-start',
+                alignItems: 'center',
+              }}
+            >
+              <Link to={homeURL}>
+                <img
+                  data-test-id="tab-logo-with-text"
+                  src={logoWhite}
+                  style={{ height: 40 }}
+                />
+              </Link>
+            </div>
+          </div>
+
+          <MoneyRaisedDisplay color="inherit" />
+        </Toolbar>
+      </AppBar>
+      <div className={cx.titleSection}>
+        <div className={cx.halfScreenLeft} data-test-id="title-wrapper">
+          <Typography variant="h1" color="primary">
+            {title}
+          </Typography>
+          <Typography className={cx.subtitle}>{subtitle}</Typography>
+          <InstallButton
+            className={cx.buttonStyles}
+            color="secondary"
+            size="medium"
+            onBeforeInstall={() => {
+              localStorageMgr.setItem(STORAGE_NEW_USER_IS_TAB_V4_BETA, 'true')
+              localStorageMgr.setItem(STORAGE_NEW_USER_CAUSE_ID, causeId)
+            }}
+            onUnsupportedBrowserInstallClick={() => {
+              setShowUnsupportedBrowserMessage(true)
+            }}
+          />
+        </div>
+        <div className={cx.halfScreenRight}>
+          <img src={headerImg}></img>
+        </div>
+        <div className={cx.wave}>
+          <Wave />
+        </div>
+      </div>
+      <div className={cx.waveMobile}>
+        <Wave />
+      </div>
+      <UnsupportedBrowserDialog
+        open={showUnsupportedBrowserMessage}
+        onClose={() => {
+          setShowUnsupportedBrowserMessage(false)
+        }}
+      />
+    </div>
+  )
+}
+Landing.propTypes = {
+  location: PropTypes.shape({
+    pathname: PropTypes.string.isRequired,
+  }),
+  pageContext: PropTypes.shape({
+    referrer: PropTypes.shape({
+      id: PropTypes.number,
+    }),
+  }),
+  causeData: PropTypes.shape({
+    causeId: PropTypes.string,
+    canonicalURL: PropTypes.string,
+    title: PropTypes.string,
+    subtitle: PropTypes.string,
+    headTitle: PropTypes.string,
+    headTitleTemplate: PropTypes.string,
+    headOgTitle: PropTypes.string,
+    headOgDescription: PropTypes.string,
+    headKeywords: PropTypes.string,
+    headOgImgURLAbsolute: PropTypes.string,
+  }),
+}
+const LandingWithTheme = (props) => (
+  <ThemeProvider theme={props.theme}>
```

Comment:
lets take the ThemProvider and CssBaseline out of the component and put it in the parent component, so we don't have to define it in each section of the homepage

Reply:
I will probably wait for your PR to land before doing this, I'll address the other comments first.